### ISSUE TITLE
Question about Fig6

### ISSUE THREAD

**Issue by CleyLyChen** [ArjunPanickssery/self_recognition#1]
Hi, great work!
I am reading your paper, but i dont know why the self-recognition score and self-preference score can be large than 1 or less than 0.
Best regards.

**Comment by ArjunPanickssery** [comment]
You're right that the score can't be more than 1 or less than 0. The figure should be reworked; its best-fit plot ends up going outside the boundaries even though none of the data does.
